project/graph_utils.py

```diff
@@ -0,0 +1,24 @@
+import cfpq_data
+from collections import namedtuple
+from typing import Tuple
+import networkx
```

Comment:
Импортировать `networkx` лучше следующим образом
```suggestion
import networkx as nx
```